tests/e2e/features/presentationViewer.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Markdown presentation viewer
+  As a user
+  I want to view the markdown in the presentation format
+  So that I can view the markdown documents in the presentation format
+  
+  Background:
+    Given the user has logged in with username "admin" and password "admin"
+    
+  
+  Scenario: Open Markdown file and verify rendering
+    Given the user opens a markdown file "index1.md"
+    When the file is rendered
+    Then the content should be displayed in a presentation viewer
+    And all tags should be rendered correctly (e.g., # -> <h1>)
+
+
+  Scenario: Verify slide content and navigation
+    Given the user opens a markdown file with multiple slides "slides.md"
```

Comment:
We need to create or upload the markdown file.

Reply:
I guess that would be part of background since it applies to all scenarios...